packages/office-ui-fabric-react/src/components/__snapshots__/Checkbox.Other.Example.tsx.shot

```diff
@@ -1117,7 +1117,6 @@ exports[`Component Examples renders Checkbox.Other.Example.tsx correctly 1`] = `
               height: 10px;
               left: 4px;
               position: absolute;
-              right: 0px;
```

Comment:
Looks like the primary change to the control is the removal of the right value. Is this necessary? Or should we expect the snapshot to be a zero change?

Reply:
He commented about that [here](https://github.com/OfficeDev/office-ui-fabric-react/pull/10170#issuecomment-522155358)